definitions/npm/prismic.io_v3.x.x/flow_>=v0.28.x/prismic.io_v3.x.x.js

```diff
@@ -1,5 +1,5 @@
 declare module 'prismic.io' {
-  declare type linkResolver = (...args: any) => any;
+  declare type LinkResolver = (...args: any) => any;
```

Comment:
@jeffmo I've improved the `LinkResolver` definition by digging into the source and fishing out the actual implementation details. I've annotated the type with the expected inputs/outputs.

@thejameskyle good to go? :)